01-python/safety/firestation.py

```diff
@@ -1,3 +1,37 @@
-def draw_firestation():
-    print("fire station not found")
+def draw_firestation(trucks=1):
+    extra_trucks = trucks-1
+    print("      ________    ")
+    print("     /        \   ")
+    print("    /          \  ")
+    print("   /            \ ")
+    print("  /______________\ ")
+    print("  |              | ")
+    print("  | Volunteer    | ")
+    print("  | Fire         | ")
+    print("  | Department   | ")
+    if trucks == 0:
+      print("  |              | ")
+      print("  |              | ") 
+      print("  |              | ") 
+      print("  |              | ") 
+      print("  |              | ")
+      print("  |              | ") 
+      print("  |______________| ") 
+    elif trucks == 1:
```

Comment:
this is a bit redundant - you can do:
```suggestion
    else:
        print("  |              |"+ extra_trucks*"______________")
        print("  |              " + extra_trucks*"              " + "| ")
        print("  |   _n____n_   " + extra_trucks*"   _n____n_   " + "| ")
        print("  |   |______|   " + extra_trucks*"   |______|   " + "| ")
        print("  |   |      |   " + extra_trucks*"   |      |   " + "| ")
        print("  |   |      |   " + extra_trucks*"   |      |   " + "| ")
        print("  |   |_|__|_|   " + extra_trucks*"   |_|__|_|   " + "| ")
        print("  |____U____U____" + extra_trucks*"____U____U____" + "| ")
`` `

Reply:
since extra_trucks will be 0 when trucks is 1